packages/boxel-ui/addon/src/components/filter-list/index.gts

```diff
@@ -0,0 +1,63 @@
+import { fn } from '@ember/helper';
+import { on } from '@ember/modifier';
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+
+import { cn, eq } from '../../helpers.ts';
+
+export type Filter = {
+  displayName: string;
+  query: any;
+};
+
+interface Signature {
+  Args: {
+    activeFilter?: Filter;
+    filters: Filter[];
+    onClickFilter?: (filter: Filter) => void;
```

Comment:
Maybe `onChanged`? I tried it out with keyboard and it does work as expected because you rightly used buttons for the filters.